settings.gradle

```diff
@@ -19,6 +19,8 @@ include ":grpc-all"
 include ":grpc-alts"
 include ":grpc-benchmarks"
 include ":grpc-services"
+include ":grpc-servlet"
+include ":grpc-servlet-testing"
```

Comment:
What is this? I don't see it.

Reply:
see line 46
```
+ project(':grpc-servlet-testing').projectDir = "$rootDir/servlet/interop-testing" as File
```

and `servlet/interop-testing/build.gradle`

Reply:
Ah, I see. Let's name that `grpc-servlet-interop-testing`.

Reply:
Done.